src/services/near.js

```diff
@@ -0,0 +1,16 @@
+import { connect, keyStores, WalletConnection } from "near-api-js";
```

Comment:
I think removing the `async/await` stuff from setting up the `Near` and WalletConnection` object is better in this case